src/webhook_launcher/app/models.py

```diff
@@ -190,6 +190,18 @@ def matches(self, proj_name):
                 return True
         return False
 
+    @classmethod
+    def get_matching(cls, name, apiurl):
+        # Search all Projects for a match
+        # Exception raised if more than one match found
+        found=None
+        for project in Project.objects.filter(obs__apiurl=apiurl):
+            if project.matches(name):
+                if found is not None:
+                    raise MultipleObjectsReturned("Project %s matches both %s and %s (possibly more)" %
+                                                  (name, found.name, project.name))
+                found = project
+        return found
```

Comment:
oh - and it is not checkable and has no defined meaning if 2 Projects claim a project name